docs/learning/Provenence-in-iModels.md

```diff
@@ -0,0 +1,57 @@
+# Provenence in iModels
+
+## What is meant by "Provenence in iModels"
+
+Provenence in iModels concerns the ability to trace an element in an iModel back to it's native, external source. Several classes are available in the BisCore schema version (> version 01.00.13) for modeling the sources-of-data of an iModel, to provide provenance information for Elements in the iModel. A RepositoryLink is a URL and it roughly maps to an external source file such as MicroStation _.dgn file or an IFC_.ifc file. The [ExternalSource](#ExternalSource) is used for external source files which are divided into smaller models. Models within a MicroStation \*.dgn are the primary use case for ExternalSource.
```

Comment:
Do you want to link to the BIS documentation for the classes referenced here?

Reply:
Following up on that comment, it seems like some of the descriptions of the entity classes below may be better suited for the BIS docs themselves, https://www.itwinjs.org/bis/domains/biscore.ecschema/#externalsourceattachment.

Reply:
@calebmshafer I was simply attempting to reference the other sections in the same document though I do agree it would be more valuable to reference the documentation for the classes.  I also see that this information may be better suited for the schema documentation too.  How do we resolve this? Should I remove this entire "provenence-in-imodels.md" from this folder and integrate this text into the schema.xml itself?

Reply:
I think the high level overview (`What is meant by "Provenance in iModels"`) still has value being here, separate from the schema docs.  Not sure if it should have it's own page or be rolled in somewhere else though, maybe just leave it in the separate file for now.

Reply:
> I think the high level overview (`What is meant by "Provenance in iModels"`) still has value being here, separate from the schema docs. Not sure if it should have it's own page or be rolled in somewhere else though, maybe just leave it in the separate file for now.

@calebmshafer @ColinKerr moved all the class by class information out of "provenance in iModels" but left "high level overview".  I will augment this article w/ CMAP image, ECSQL query in future PR.  Will also move class by class information into remarks.md for biscore in future PR.